Makefile

```diff
@@ -1471,7 +1473,7 @@ ifeq ($(PLATFORM), OS_SOLARIS)
 	ROCKSDBJNILIB = librocksdbjni-solaris$(ARCH).so
 	ROCKSDB_JAR = rocksdbjni-$(ROCKSDB_MAJOR).$(ROCKSDB_MINOR).$(ROCKSDB_PATCH)-solaris$(ARCH).jar
 	JAVA_INCLUDE = -I$(JAVA_HOME)/include/ -I$(JAVA_HOME)/include/solaris
-	SHA256_CMD = digest -a sha256
+	SHA1_CMD = digest -a sha1
```

Comment:
@sagar0 Well this is only for the purpose of verifying the checksum of a downloaded file. I don't see any issue with using sha1 for that. Also I don't see sha1 going away anytime soon.